gradio/cli/commands/components/docs.py

```diff
@@ -77,67 +77,118 @@ def _docs(
 
         with open(_component_dir / "pyproject.toml") as f:
             data = toml.loads(f.read())
-        with open(_demo_path) as f:
-            demo = f.read()
 
         name = get_deep(data, ["project", "name"])
 
         if not isinstance(name, str):
             raise ValueError("Name not found in pyproject.toml")
 
-        pypi_exists = requests.get(f"https://pypi.org/pypi/{name}/json").status_code
+    run_command(
```

Comment:
The warning about python 3.9 needs to be printed with `live` otherwise it's out of order:

<img width="728" alt="image" src="https://github.com/gradio-app/gradio/assets/41651716/b190f9f1-f82a-4df5-9e6f-76ac1fb2928f">

Reply:
I kinda wanted it to be not in the live panel because there isn't actually an issue with the doc gen, just a note. But I can put it at the bottom with some space. 

Will update the message.

Reply:
Looks great! My bad, I think we should remove the backticks around --no-generate-docs since they don't change. how it renders